src/main/java/offeneBibel/osisExporter/ObOsisGeneratorVisitor.java

```diff
@@ -173,9 +180,26 @@ else if(node.getNodeType() == ObAstNode.NodeType.verse) {
                 m_currentFassungContainsVerses = true;
                 m_skipVerse = false;
                 m_verseTag = m_verseTagStart + verse.getNumber();
-                m_currentFassung += "<verse osisID=\"" + m_verseTag + "\" sID=\"" + m_verseTag + "\"/>";
+                m_currentFassung.append("<verse osisID=\"" + m_verseTag + "\" sID=\"" + m_verseTag + "\"/>");
+                if (m_inlineVersStatus) {
+                    ObVerseStatus status = verse.getStatus();
+                    ObVerseStatus leseStatus = verse.getStatus(FassungType.lesefassung);
+                    ObVerseStatus studienStatus = verse.getStatus(FassungType.studienfassung);
+                    String verseStatus;
+                    if (leseStatus == studienStatus || status == studienStatus) {
```

Comment:
in der Studienfassung erscheint immer nur der Studienfassungsstatus (`if (status == studienStatus)`). In der Lesefassung erscheinen beide Stati wenn abweichend.

Mein Hintergrund war, dass bei einer "in Bearbeitung" befindlichen Lesefassung durchaus relevant sein kann, ob die zugrundeliegende Studienfassung roh oder final war.